pkg/users/users.js

```diff
@@ -165,11 +165,31 @@
         console.warn("Unexpected error when getting logged in accounts", err);
     }
 
+    // shadow-utils passwd supports an --all flag which is lacking on RHEL and
+    // stable Fedora releases. Available at least on Fedora since
+    // shadow-utils-4.14.0-5.fc40 (currently known as rawhide).
+    const locked_users_map = {};
+    try {
+        const locked_statuses = await cockpit.spawn(["passwd", "-S", "--all"], { superuser: "require", err: "message", environ: ["LC_ALL=C"] });
+        // Slice off the last empty line
+        for (const line of locked_statuses.trim().split('\n')) {
+            const username = line.split(" ")[0];
+            const status = line.split(" ")[1];
+            locked_users_map[username] = status == "L";
+        }
+    } catch (err) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19976-20240209-110048-72f90fa8-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)